modules/http4s/src/test/scala/pureconfig/modules/http4s/Http4sTest.scala

```diff
@@ -12,6 +12,6 @@ class Http4sTest extends BaseSuite {
   "reading the uri config" should "parse the uri" in {
```

Comment:
Please add a case which tests the unhappy path of a bad URI.

Reply:
will do

Reply:
@jcranky, Thanks!